functions/src/chat.ts

```diff
@@ -0,0 +1,25 @@
+import type { DocumentReference } from "firebase-admin/firestore";
+import { db } from "./app";
+import { getChatRoomCollection, getChatRoomMessagesCollection, getLobbyChatCollection } from "./firestore-collections";
+import { Lobby } from "./firestore-types/lobby";
+
+export function deleteChatRooms(lobbyDoc: DocumentReference<Lobby>) {
+  db.runTransaction(async (transaction) => {
+    const lobbyDocRef = (await transaction.get(lobbyDoc)).ref;
+    const chatRooms = await getChatRoomCollection(lobbyDocRef).listDocuments();
+
+    // get prompts answer then delete chatrooms
+    const lobbyChatCollection = getLobbyChatCollection(lobbyDocRef);
```

Comment:
as we discussed lets place this into the lobby doc and put a isPromptAnswer (or smething like that) property on chat message